doc/api/http2.md

```diff
@@ -722,14 +722,10 @@ req.on('response', (headers) => {
 });
 ```
 
-When set, the `options.getTrailers()` function is called immediately after
-queuing the last chunk of payload data to be sent. The callback is passed a
-single object (with a `null` prototype) that the listener may use to specify
-the trailing header fields to send to the peer.
-
-The HTTP/1 specification forbids trailers from containing HTTP/2 pseudo-header
-fields (e.g. `':method'`, `':path'`, etc). An `'error'` event will be emitted
-if the `getTrailers` callback attempts to set such header fields.
+When the `options.hasTrailers` option is set, the `'trailers-ready'` event
```

Comment:
Our docs currently list 21 event names that consist of more than one word; All of them use `lowerCamelCase`, so it might be good to be consistent with that?